src/bpfd.h

```diff
@@ -35,11 +37,18 @@
 	if (!sscanf(tok, "%u ", &var))		\
 		goto invalid_command;
 
+#define PARSE_UINT64(var)			\
+	tok = strtok(NULL, " ");		\
+	if (!tok)				\
+		goto invalid_command;		\
+	if (!sscanf(tok, "%"SCNu64" ", &var))	\
+		goto invalid_command;
+
 #define PARSE_ULL(var)				\
 	tok = strtok(NULL, " ");		\
 	if (!tok)				\
 		goto invalid_command;		\
-	if (!sscanf(tok, "%llu ", &var))		\
```

Comment:
remove extra tab

Reply:
yep, its superflous

Reply:
Oh ok I got confused. Leave the change in, its fine. Could you squash everything though, "uprobes support" can be one commit.

Reply:
Done.